src/ui/viewmodels/TriggerConditionViewModel.cpp

```diff
@@ -219,6 +222,15 @@ void TriggerConditionViewModel::SerializeAppendOperand(std::string& sBuffer, Tri
             sBuffer.push_back('~');
             break;
 
+        case TriggerOperandType::Recall:
+        {
+            std::string recall = "{recall}";
+            for (size_t i = 0; i < recall.length(); i++) {
+                sBuffer.push_back(recall.at(i));
```

Comment:
Appending using this method for better clarity when reading the code, but perhaps it would be better to push the individual characters?

Reply:
Just use append:
```
sBuffer.append("{recall}");
```